src/index.js

```diff
@@ -5,32 +5,32 @@ if (!TESTING)
 
 const { EventEmitter } = require('events');
 
-
 class Connection extends EventEmitter {
 
-  constructor(url) {
+  constructor(url, signalCb) {
```

Comment:
Code update looks and works great. The only missing part I am seeing here is documentation. Three things  that need to be updated:
- `signalCB` in constructor and what are the consequences of not passing it (i.e. when `signalCb == undefined`)
- `.ready()` does not accept `timeout` argument anymore
- what is `url` in constructor used for (ie when and why would we want to point UI to a different chaperone_url)

Reply:
👍